demisto_sdk/commands/init/initiator.py

```diff
@@ -627,3 +635,28 @@ def copy_demistotmock(self):
                 shutil.copy(f'{self.configuration.env_dir}/Tests/demistomock/demistomock.py', self.full_output_path)
             except Exception as err:
                 print_v(f'Could not copy demistomock: {str(err)}')
+
+    def get_remote_templates(self, files_list):
+        """
+        Downloading the object related template-files and saving them in the output path.
+        Args:
+            files_list: List of files to download.
+        Returns:
+            bool. True if the files were downloaded and saved successfully, False otherwise.
+        """
+        if self.is_integration:
+            path = os.path.join('Packs', 'HelloWorld', 'Integrations', 'HelloWorld')
+            os.mkdir(os.path.join(self.full_output_path, self.TEST_DATA_DIR))
+        else:
+            path = os.path.join('Packs', 'HelloWorld', 'Scripts', 'HelloWorldScript')
```

Comment:
don't we want to create a test dir for scripts?

Reply:
the current template doesn't include one and most scripts don't have test files so I don't think it is necessary

Reply:
it is not needed. `HelloWorld` pack doesnt have a test_data dir inside the HelloWorldScript directory. Not sure we want to create an empty folder.